src/main/java/Dross.java

```diff
@@ -0,0 +1,85 @@
+import java.util.Scanner;
+
+public class Dross {
+
+    //Size 100 array tasklist for Dross
+    private static List drossTaskList = new List();
+    //Method that prints line using "_" char
+    public static void printLine() {
+        for (int i = 0; i < 55; i++) {
+            System.out.print("_");
+        }
+        System.out.println();
+    }
+
+    //Method that echoes back whatever user typed in provide that it is not "bye"
+    public static void echo(){
```

Comment:
Although `echo` here is a verb, perhaps it could be named to be more descriptive? An alternative name could be `echoUserInput`.